packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -56,6 +56,13 @@ const ACTIONS = [
 ]
 
 const INDIVIDUAL_ACTIONS = [
+  {
+    collapsed: true,
+    disabled: ({ url }) => url === undefined,
+    handler: ({ url }) => navigator.clipboard.writeText(url),
```

Comment:
Make it safer and more homogeneous with the rest of the code by using `copy-to-clipboard` instead.